.github/workflows/ci.yaml

```
@@ -97,7 +98,7 @@ jobs:
   format:
     name: Format
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    timeout-minutes: 5
```

```suggestion
    timeout-minutes: 10
```
This _should_ be fast, but with private runners and CPU RNG I'd go a little higher here.